.gitignore

```diff
@@ -15,4 +15,25 @@ build2
 gh-pages
 .tags*
 .stdafx.c++.pch
-meson-build
\ No newline at end of file
+meson-build
+config.log
+config.h
+config.status
+*.pc
+gtkmm-plplot.spec
+libtool
+.gitignore
+Makefile
+stamp-h1
+tests/.deps
+docs/Doxyfile
+docs/html
+*.lo
+*.la
+docs/doxyfile.stamp
+tests/.libs
+tests/test[0-9]*
+gtkmm-plplot/.deps
+gtkmm-plplot/stamp-h2
+gtkmm-plplot/gtkmm-plplotconfig.h
+gtkmm-plplot/.libs
```

Comment:
The vast majority of these are only relevant for Autotools builds, and can be removed now